app/src/main/java/com/codingblocks/cbonlineapp/adapters/ChoicesAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package com.codingblocks.cbonlineapp.adapters
+
+import android.content.Context
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.codingblocks.cbonlineapp.R
+import com.codingblocks.onlineapi.models.Question
+import kotlinx.android.synthetic.main.choice_numbers_layout.view.*
+import kotlinx.android.synthetic.main.item_carousel.view.*
+
+class ChoicesAdapter(private val listener:ChoiceClickListener) : RecyclerView.Adapter<ChoicesAdapter.MyViewHolder>() {
+
+    lateinit var context : Context
+    private var numbers = ArrayList<Int>()
+    private var question = ArrayList<Question>()
+
+    fun setdata(question: ArrayList<Question>, numbers:ArrayList<Int>){
+        this.numbers = numbers
+        this.question = question
+        notifyDataSetChanged()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyViewHolder {
+        context = parent.context
```

Comment:
not needed

Reply:
this context will be required in bindViewHolder